Python/bytecodes.c

```diff
@@ -1104,8 +1104,10 @@ dummy_func(
                 (Py_TYPE(receiver_o) == &PyGen_Type || Py_TYPE(receiver_o) == &PyCoro_Type) &&
                 ((PyGenObject *)receiver_o)->gi_frame_state < FRAME_EXECUTING)
             {
+                _PyInterpreterFrame *gen_frame;
```

Comment:
I have a feeling you'll need to fix the specializations too. But right now they aren't enabled, so no way to test them until Matt re-enables them :(.

Reply:
PTAL at GH-127426 and https://github.com/python/cpython/pull/127426#issuecomment-2508765289 now. I think we're good for the specializations.